studio.assetmanager.ams.yaml

```diff
@@ -0,0 +1,77 @@
+app-id: studio.assetmanager.ams
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: ams
+finish-args:
+  # X11 performance
+  - --share=ipc
+  # Access to network
+  - --share=network
+  # Access to X11
+  - --socket=x11
+  # Access to wayland
+  - --socket=wayland
+  # Audio Access
+  - --socket=pulseaudio
+  # Access to all files
+  - --filesystem=host
+  # Access to OpenGL
+  - --device=dri
+  # Use same mouse cursors as host
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --talk-name=org.freedesktop.secrets
+
+rename-desktop-file: asset-manager-studio.desktop
+rename-icon: asset-manager-studio
+modules:
+  - name: ams-desktop
+    buildsystem: simple
+    build-commands:
+      # Extract .deb & cleanup
+      - ar -x asset-manager-studio*.deb
+      - rm -f asset-manager-studio*.deb
+      - tar -xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
```

Comment:
```suggestion
```

Reply:
This is common in many of the other published flatpacs using the extraction of the deb file.   
See for example: 
https://github.com/flathub/com.bitwarden.desktop/blob/3603fbffe8c50fa4ea3e2c6e34d6f2545740507b/com.bitwarden.desktop.yaml#L31-L34

Is this no longer needed?   I looked at over a dozen electron from deb that are actually published to figure out what I needed to do.  ;-)

Reply:
because I never reviewed that package. it's not because they do it that it is right.

two wrongs don't make a right.